apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/config/SwiftAuthenticationModule.java

```diff
@@ -153,7 +173,7 @@ private URI getURI(String headerValue) {
       public AdaptTempAuthResponseToAccess setContext(HttpRequest request) {
          String host = request.getEndpoint().getHost();
          this.host = host;
-         this.username = request.getFirstHeaderOrNull(STORAGE_USER);
+	 this.username = request.getFirstHeaderOrNull(SwiftAuthenticationModule.getIdentityHeaderName());
```

Comment:
You can inject the header variable in this class too, to avoid dealing with static methods.